src/main/scala/io/sdkman/changelogs/MavenMigration.scala

```diff
@@ -105,4 +105,14 @@ class MavenMigration {
       .asCandidateDefault()
   }
 
+  @ChangeSet(order = "009", id = "009-add_maven_4.0.0-alpha-1-SNAPSHOT", author = "helpermethod")
+  def migration009(implicit db: MongoDatabase) = {
+    Version(
+      "maven",
+      "4.0.0-alpha-1-SNAPSHOT",
```

Comment:
Mind if we drop `-SNAPSHOT` from the version? I think it might exceed column bounds and it doesn't really add anything to the uniqueness of the version.

Reply:
Good point!